eng/Versions.props

```diff
@@ -174,10 +174,6 @@
     <MicrosoftTestPlatformCLIPackageVersion>17.14.0-preview-25157-01</MicrosoftTestPlatformCLIPackageVersion>
     <MicrosoftTestPlatformBuildPackageVersion>17.14.0-preview-25157-01</MicrosoftTestPlatformBuildPackageVersion>
   </PropertyGroup>
-  <PropertyGroup>
-    <!-- Dependencies from https://github.com/Microsoft/testfx -->
-    <MicrosoftTestingPlatformVersion>1.5.0-preview.24602.2</MicrosoftTestingPlatformVersion>
-  </PropertyGroup>
```

Comment:
This was overwriting https://github.com/dotnet/sdk/blob/2fc15375015374315698cb7679c03fccf9ca3b21/eng/Versions.props#L32-L33

And it seems Arcade was only updating the first one.

That version is too old.